gef.py

```diff
@@ -802,10 +802,9 @@ class MallocStateStruct:
 
     def __init__(self, addr):
         try:
-            self.__addr = parse_address("&{}".format(addr))
+            pattern = "&0x{:x}".format(addr) if is_hex(addr) else "{}".format(addr)
```

Comment:
```suggestion
            pattern = "&{:#x}".format(int(addr, 16)) if is_hex(addr) else "{}".format(addr)
```

Reply:
shouldn't addr already be an int? Why call `int( , 16)` on it?

Reply:
iirc `addr` is a string for `MallocStateStruct`. This is one of the reasons why the "Type Hints" PR is very much overdue I guess :D

Reply:
Yeah. Let's get that in first before we worry about the rest.